analyzers/golang/testdata/gomod/.fossa.yml

```diff
@@ -0,0 +1,14 @@
+# Generated by FOSSA CLI (https://github.com/fossas/fossa-cli)
```

Comment:
I don't think you meant to keep this file here.

Reply:
This is intentional, although not used in the test. Running `fossa analyze` with this configuration file will reproduce the bug.

Reply:
Ok, that's fine, I don't feel strongly about removing it.